frontend/src/components/DocumentChip.vue

```diff
@@ -128,7 +128,7 @@ export default {
   },
   methods: {
     deleteFile(documentData) {
-      return this.$store.dispatch(`edx/deleteFile`, documentData);
```

Comment:
Hmmm - wondering if we had these in place for a reason? (Instead of single quotes)